gemfiles/rails_7.1.gemfile

```diff
@@ -13,5 +13,6 @@ gem "activerecord-jdbcsqlite3-adapter", platform: :jruby
 gem "sequel"
 gem "evt"
 gem "async"
+gem "libev_scheduler"
```

Comment:
looks like this line snuck in accidentally?

Reply:
yes, thanks! some CI vs Local development weirdness...